xml/opensuse_update.xml

```diff
@@ -419,8 +419,8 @@ udev           252M  124K  252M   1% /dev
        <listitem>
         <para>
          This section allows you to change the <guimenu>Primary
-         Language</guimenu>primary language and configure additional
-         <guimenu>Secontry Languages.</guimenu>. Optionally, you can adjust the
+         Language</guimenu> and configure additional
+         <guimenu>Secontry Languages.</guimenu> Optionally, you can adjust the
```

Comment:
Hi @victorhck, thank you for the PR! "Secontry Languages" looks like a misspelling, should that be "Secondary Languages"?

Reply:
ups! I though it was a right term, that's why I kept the term...

Reply:
I know, right? It could have been real :)  Thanks for fixing it.